gamebook/fight_scene.py

```diff
@@ -0,0 +1,45 @@
+from gamebook.scene import Scene
+import gamebook.important_strings
+
+
+class FightScene(Scene):
+    """
+        A scene containing a battle with a monster
+
+        arguments:
+
+        next_scenes - a list of two NextScene objects, 1st is the victory scene
+        and 2nd is the the defeat scene
+
+        enemy - a monster instance
+
+    """
+    def __init__(self, name, desc, next_scenes, enemy):
+        self.name = name
+        self.desc = desc
+        self.next_scenes = {"win": next_scenes[0], "loss": next_scenes[1]}
+        self.enemy = enemy
+
+    def get_name(self):
+        return super().get_name()
+
+    def show_desc(self):
+        return super().show_desc()
+
+    def show_options(self):
+        return super().show_options()
+
+    def show_monster(self):
```

Comment:
I would generaly decouple the presentation (`show`) methods from the logic. So here for example it would be `get_monster` and then a presentation class would get that monster object and present it however you like (`MonsterPresentation -> ASCIIMonster/WebMonster etc...)

Reply:
oh got it 👍

Reply:
what about putting that function into the monster class instead and have produce a string that the [output](https://github.com/arielofer/gamebook-maker/blob/master/gamebook/output.py) function can get? it would be just like any other `show` function right?

Reply:
Can you show a quick example? (Also this is for a different PR by the way).

Reply:
monster.py:
```
    def show_monster(self):
        data_string = ""
        data_string += self.name + "\n"
        for trait in self.traits:
            data_string + f"trait name: {trait.name}, value: {trait.value}\n"

        return data_string
```
-----------------------------------------
let's say that I want to print the monster before a battle I would do something like this:
`self.output_instance.output(monster.show_monster())`

Reply:
@arielofer see if you can implement something like https://refactoring.guru/design-patterns/strategy in a next or for the output stuff.